### PR TITLE
Aml 4038 in account sign agreement link

### PR DESCRIPTION
**Pull request by Howesy77**

### CODE REVIEW

File: src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs [hunk 1]
```diff
@@ -128,6 +129,13 @@ public async Task<ActionResult> SignAgreement(GetEmployerAgreementRequest reques
             var response = await _mediator.SendAsync(request);
             var viewModel = _mapper.Map<GetEmployerAgreementResponse, EmployerAgreementViewModel>(response);
 
+            var legalentities = await _mediator.SendAsync(new GetAccountLegalEntitiesByHashedAccountIdRequest
+            {
+                HashedAccountId = request.HashedAccountId
+            });
+
+            viewModel.LegalEntitiesCount = legalentities.LegalEntities.Count;
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs]: With my performance hat on...this is a very inefficient way of getting the legal entity count.  Could you write a specific query to return just the count?

Comment (Howesy77) [reply src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs]: Yes can do

Comment (Howesy77) [reply src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs]: How about using the AccountStats query?

Comment (MichaelYoung1981) [reply src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs]: Does too much unfortunately.

Comment (Howesy77) [reply src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs]: Ok changes made

**Pull request merged**